server/auth/types/basic/basic_auth.ts

```diff
@@ -94,22 +104,37 @@ export class BasicAuthentication {
       if (!cookie) {
         return response.unauthorized();
       }
+
+      let headers = {};
+
       // set cookie to extend ttl
       cookie.expiryTime = Date.now() + this.config.cookie.ttl;
       this.sessionStorageFactory.asScoped(request).set(cookie);
-      
+
       // pass credentials to request to Elasticsearch
       const credentials = cookie.credentials;
+      assign(headers, { authorization: credentials.authHeaderValue });
```

Comment:
do we need to test null for credentials?

Reply:
changed to optional chaining:
```
Object.assign(headers, { authorization: cookie.credentials?.authHeaderValue });
```